components/game/game.model.ts

```diff
@@ -424,6 +456,19 @@ export class Game extends Model {
 		);
 	}
 
+	async $inviteCollaborator(username: string, role: typeof GameCollaborator.prototype.role) {
```

Comment:
I don't see much of a difference between this and GameCollaborator.$invite. You can always make a new collaborator, fill in the username/role fields, and then save it, right?